packages/redux-saga/package.json

```diff
@@ -29,7 +29,7 @@
   "bundlesize": [
     {
       "path": "./dist/redux-saga.umd.min.js",
-      "maxSize": "6.6 Kb"
+      "maxSize": "6.62 Kb"
```

Comment:
q: any reason behind the change here? as far as i can see there are no changes in this PR that would affect the bundlesize

Reply:
I'm assuming upgrading node created different outputs for some reason?  I can do a diff on the two bundles and see what has changed.

Reply:
Running a production build on v12.22.10 I'm getting the same exact bundle size as I am on v16.  My guess is if we fixed the dslint issue on master we would still have a failing test for bundlesize.

My vote is to let this change through so we can focus on other cleanup issues.

Reply:
For reference - this is the last commit that has executed this check: https://github.com/redux-saga/redux-saga/commit/a8c968b77243f59fbe672ca56691761664e060bf . It's somewhat weird as it shouldn't even rely on other CI checks... 🤔 as in, it should execute regardless of the status of those other CI checks

q: how did you check that this value has to be adjusted in the first place? no CI checks were executed in this PR

Reply:
It seems like once our CI has a single failure all the jobs bail.

> q: how did you check that this value has to be adjusted in the first place? no CI checks were executed in this PR

https://github.com/neurosnap/redux-saga/runs/5341759335?check_suite_focus=true